src/main/webapp/app/lecture/lecture-update.component.html

```diff
@@ -1,145 +1,140 @@
 <div class="row justify-content-center">
     <div class="col-md-8">
-        @if (isShowingWizardMode) {
-            <jhi-lecture-update-wizard
-                [toggleModeFunction]="toggleModeFunction"
-                [saveLectureFunction]="saveLectureFunction"
-                [validateDatesFunction]="onDatesValuesChanged"
-                [lecture]="lecture"
-                [isSaving]="isSaving"
-            />
-        }
-        @if (!isShowingWizardMode) {
-            <form name="editForm" role="form" (ngSubmit)="save()" #editForm="ngForm">
-                <div class="d-flex">
-                    <div class="flex-grow-1">
-                        <div class="d-flex align-items-center">
-                            <h2 id="jhi-lecture-heading" jhiTranslate="artemisApp.lecture.home.createOrEditLabel"></h2>
-                            <jhi-documentation-button [type]="documentationType" />
-                        </div>
-                    </div>
-                    <div class="flex-grow-1 ms-3">
-                        <div class="guided-mode-button">
-                            <button type="button" id="guided.mode" class="btn btn-primary" (click)="toggleWizardMode()">
-                                <fa-icon [icon]="faHandShakeAngle" />&nbsp;<span jhiTranslate="artemisApp.lecture.home.switchToGuidedModeLabel"></span>
-                            </button>
-                        </div>
+        <form name="editForm" role="form" (ngSubmit)="save()" #editForm="ngForm">
```

Comment:
The div is rather hard to read here

* Removed wizard mode
* moved course info further within title section
* added `jhi-form-status-bar`
* removed duplicated datepicker code and using `jhi-lecture-update-period` instead
* added uniform h2 and span to each section for the status bar navigation to work

Reply:
Easy to read if you use the `Hide Whitespace` functionality, thanks for pointing it out @rabeatwork 😄